buildpipeline/DotNet-CoreClr-Trusted-Linux-Crossbuild.json

```diff
@@ -193,7 +193,25 @@
       },
       "inputs": {
         "filename": "docker",
-        "arguments": "run --rm $(DockerCommonRunArgs) ./publish-packages.sh -AzureAccount=$(CloudDropAccountName) -AzureToken=$(CloudDropAccessToken) -BuildType=$(PB_BuildType)  -BuildArch=$(Architecture) -Container=$(Label) -distroRid=$(Rid) -- /p:OverwriteOnPublish=true",
```

Comment:
Is `$(Label)` for the container name going to be unique enough? What does it get set to?

Reply:
Also you don't appear to be passing the RelativePath here that that will include the correct information about the build.

Reply:
Label is something like `coreclr-beta-20170918-02`. It uniquely identifies a pipebuild, and all legs of that pipebuild share the same label - this is what's used by dotnet-trusted-publish to find & download the packages from the current build

Reply:
My expectation is that this would now be a fixed container name and we would instead be using RelativePath to segregate the builds.